backend/test/getCurrentExpense.test.ts

```diff
@@ -0,0 +1,85 @@
+import { PostgreSqlContainer, type StartedPostgreSqlContainer } from '@testcontainers/postgresql';
+import type { FastifyInstance } from 'fastify';
+import { afterAll, beforeAll, describe, expect, test } from 'vitest';
+
+import { buildApp } from '@/app';
+
+import * as schema from '../src/db/schema';
+describe('/get_current_expense route', () => {
+  let container: StartedPostgreSqlContainer;
+  let app: FastifyInstance;
+  const testAddress = '0x004babd76a282efdd30b97c8a98b0f2e4ebb91e81b3542bfd124c086648a07af';
+  const amount = 1000;
+
+  beforeAll(async () => {
+    container = await new PostgreSqlContainer().start();
+    const connectionUri = container.getConnectionUri();
+    app = buildApp({
+      database: {
+        connectionString: connectionUri,
+      },
+      app: {
+        port: 8080,
+      },
+    });
+
+    await app.ready();
+    // Insert transfer to mock address
+
+    await app.db.insert(schema.usdcBalance).values({ address: testAddress, balance: '1000' });
+    await app.db
+      .insert(schema.usdcTransfer)
+      .values({ transferId: '1', fromAddress: testAddress, amount: amount.toString() });
```

Comment:
use the proper format for transfer ids https://github.com/keep-starknet-strange/vault/blob/3a6bad720c50a91ab9967d8958c8bc6261e8e7b8/infra/indexers/src/usdc-transfers.indexer.ts#L34